index.d.ts

```diff
@@ -75,6 +75,36 @@ declare const isStream: {
 	```
 	*/
 	transform(input: unknown): input is stream.Transform;
+
+	/**
+	@returns Returns a `string` to tell the [`type of stream`](https://nodejs.org/api/stream.html#stream_types_of_streams): `readable`, `writable`, `duplex`, `transform` or `undefined` for unknown stream type and non streams.
```

Comment:
You're writing "return" twice.

Reply:
Indeed! ca13635ef94d0b31b6af4eb023dc8aadefd4a899